Ports/python3/package.sh

```diff
@@ -36,6 +36,7 @@ configopts=(
     '--without-ensurepip'
     'ac_cv_file__dev_ptmx=no'
     'ac_cv_file__dev_ptc=no'
+    'CFLAGS=-fPIC'
```

Comment:
`CCSHARED` sounds more appropriate for this.

Reply:
fwiw, if setting this in `CCSHARED` fixes it, we should just patch it in accordingly and send the patch upstream. Many other OSes already default `CCSHARED="-fPIC" in the `configure` script.

Reply:
Sounds like a good idea to me. This was enough for me locally, but it would be great if @oskar-skog could test it as well.

Reply:
Actually, our platform support patch isn't upstream yet (i.e. nothing to update there), so we just need to add it to the existing `0002-Tweak-configure-and-configure.ac.patch`.

Reply:
I can confirm that using `CCSHARED` works.  (Tested both as option sent to configure and patched configure itself.)

Is there a correct procedure for updating a patch?

Also, while testing I found out that `0007-Set-name-of-shared-libpython.patch` also patches `configure`, *but not* `configure.ac`.   I guess I'll merge that in as well but for both `configure` and `configure.ac` to avoid confusion?

Reply:
`configure.ac` is unused as long as we don't run `autoconf` (which we should never do unless absolutely necessary, since that regenerates `configure` using host dependencies). We should probably drop the modifications to `configure.ac`.

Reply:
> Is there a correct procedure for updating a patch?

Run `./package.sh dev`, which imports all the patches as Git commits and drops you into a shell. In there, you can edit the commits as usual via `rebase`, `commit --amend`, etc. Once this is done and you exit the shell (via `Ctrl+D`) our scripts will automatically generate patches from those commits and update the ReadMe.